MarkBind/markbind
# Use js for tests

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Other, please explain: Standardisation of test / update test scripts

Resolves #829 

**What is the rationale for this request?**
Standardise scripts to reduce maintainence from 2 sets of files (bash + sh)

**What changes did you make? (Give an overview)**
- Use the existing `.sh` script as a reference, creating test / update test scripts with js
- Remove existing `win` version of CI scripts (and in dev guide)

**Is there anything you'd like reviewers to focus on?**
Any additional test cases I missed out below

**Testing instructions:**
For each category of test sites ( conversion, normal test sites, template test sites ),
- change something in the source file -> tests should fail
- change something in the source file ->  update tests -> test files update as expected

**Proposed commit message: (wrap lines at 72 characters)**
Use javascript for tests

Test and test update scripts are written using bash and shell scripts,
which increases maintainence due to having to maintain two sets of OS
dependent scripts.

Let's rewrite these scripts using javascript instead.
